modules/installation-osp-adding-bare-metal.adoc

```diff
@@ -0,0 +1,137 @@
+[id="installation-osp-adding-bare-metal_{context}"]
+= Adding bare metal compute machines to a cluster that is deployed on {rh-openstack}
+//FIXME title is essentially the assembly--mod it
+You can add bare-metal compute machines to an {product-title} cluster after you deploy it
+on {rh-openstack-first}. In this configuration, all machines are attached to an
+existing, installer-provisioned network, and traffic between control plane and
+compute machines is routed between subnets.
+
+.Prerequisites
+
+* The {rh-openstack} link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/index[Bare Metal service (Ironic)] is enabled and accessible by using the {rh-openstack} Compute API.
+
+* Bare metal is available as link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.1/html/bare_metal_provisioning/sect-configure#creating_the_bare_metal_flavor[an {rh-openstack} flavor].
+
+* You deployed a {product-title} cluster on installer-provisioned infrastructure.
+
+* Your {rh-openstack} cloud provider is configured to route traffic between the installer-created VM
+subnet and the pre-existing bare metal subnet.
+
+.Procedure
+
+. Create a file called `baremetalMachineSet.yaml`, and then add the bare metal flavor to it:
++
+.A sample bare metal MachineSet file
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
```

Comment:
This is just the MachineSet creation module example reused. Is there a specific one for this task that we want to use?